### PR TITLE
add: added all the main features

### PR DESCRIPTION
**Pull request by kumsomi**
- Clock
- Focus
- Weather
- Quote

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -36,3 +36,174 @@
...
+.greeting{
+  font-size: 3rem;
+  margin:2rem;
+}
+
+.quote{
+  padding: 1rem;
+  font-size: 3rem;
+  font-weight: 300;
+  background-color: rgba(0, 0, 0, 0.3);
```
Comment (ianuragsharma) [inline src/App.css]: better define a CSS Variable  for this color

File: src/App.css [hunk 2]
```diff
@@ -36,3 +36,174 @@
...
+.focus-input:focus::placeholder{
+  color: transparent;
+}
+.floating-btn{
+  border: 0;
+  background-color: transparent;
+}
+.floating-btn:hover{
+  background-color: rgba(247, 241, 241, 0.5);
+  /* color:black; */
```
Comment (ianuragsharma) [inline src/App.css]: remove the commented code

File: src/App.css [hunk 3]
```diff
@@ -36,3 +36,174 @@
...
+.weather {
+  position: fixed;
+  right: 1rem;
+  top: 1rem;
+  text-align: right;
+  display: flex;
+}
+.weather-details{
+  /* font-size: 3rem; */
+  /* font-family: ; */
```
Comment (ianuragsharma) [inline src/App.css]: same as above

File: src/Components/Greeting.jsx [hunk 4]
```diff
@@ -1,7 +1,35 @@
...
+import { useTime } from "../Context/TimeContext";
+
 function Greeting(){
+    const { partOfDay } = useTime();
+    
+    const [greeting, setGreeting] = useState("");
+    const username = localStorage.getItem("username");
+    const displayName = username.charAt(0).toUpperCase() + username.slice(1);
+    useEffect(() => {
+        console.log(partOfDay);
```
Comment (ianuragsharma) [inline src/Components/Greeting.jsx:11]: remove console

**Review by ianuragsharma — APPROVED** [review]
left few comments please look into it.
Other than that good to merge.
UI looks good.

File: src/App.css (hunk 2 above)
Comment (kumsomi) [reply src/App.css]: > left few comments please look into it. Other than that good to merge. UI looks good.

Thanks @ianuragsharma for reviewing. I made the necessary changes pointed out in file

**Pull request merged**